doc/random.md

```diff
@@ -65,7 +65,7 @@ Returns an unsigned 32 bit integer random number.
 <a name="torch.uniform"></a>
 ### [number] uniform([a],[b]) ###
 
-Returns a random real number according to uniform distribution on [a,b[. By default `a` is 0 and `b` is 1.
+Returns a random real number according to uniform distribution on [a,b]. By default `a` is 0 and `b` is 1.
```

Comment:
Hi,
I believe this was right as is, it is indeed an open interval in `b`, as mentioned [in the code](https://github.com/torch/torch7/blob/master/lib/TH/THRandom.c#L203).
Thanks for improving the docs !